lib/verdict/storage/base_storage.rb

```diff
@@ -45,11 +45,21 @@ def store_start_timestamp(experiment, timestamp)
         set(experiment.handle.to_s, 'started_at', timestamp.utc.strftime('%FT%TZ'))
       end
 
-      def cleanup(_scope)
-        raise NotImplementedError
+      # Deletes all assignments (and any other stored data) for the given experiment
+      def cleanup(experiment_or_scope)
+        if experiment_or_scope.is_a?(Symbol) || experiment_or_scope.is_a?(String)
+          Verdict.default_logger.warn(
+            "Passing a scope string/symbol to #{self.class}.cleanup is deprecated, " \
```

Comment:
Nit: 
```suggestion
            "Passing a scope string/symbol to #{self.class}#cleanup is deprecated, " \
```

Reply:
Thanks, I will apply this change when rebasing